src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -417,6 +417,31 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         execCond("docker rmi -f ${id}") || true
     }
 
+    def 'check lables are correctly applied to build'() {
+        given:
+        String id = 'id10'
+        temporaryFolder.newFile('Dockerfile') << """
+            FROM alpine:3.2
+        """.stripIndent()
+        buildFile << """
+            plugins {
+                id 'com.palantir.docker'
+            }
+
+            docker {
+                name '${id}'
+                labels(['test_label': 'test_value'])
```

Comment:
Should be possible to apply this without the parens or square brackets, I think.